src/components/AssetHeader/AssetActions.tsx

```diff
@@ -73,10 +76,23 @@ export const AssetActions: React.FC<AssetActionProps> = ({ assetId, accountId, c
       flex={1}
     >
       <Flex direction='row' gap={2} flexWrap='wrap'>
+        <Button
+          data-test='asset-action-trade'
+          display={{ base: 'auto', md: 'none' }}
+          flex={{ base: 1, md: 'auto' }}
+          leftIcon={<IoSwapVertical />}
+          size='sm-multiline'
+          width={{ base: '100%', md: 'auto' }}
+          isDisabled={!isValidChainId}
+        >
+          <Link to={tradeAssetLink}>
+            {translate('assets.assetCards.assetActions.tradeAsset', { symbol: asset.symbol })}
```

Comment:
This is prob gonna burn us for assets with really long symbols. Might just stick with ‘Trade’. I know we have the ability to wrap the buttons now but with this layout of buttons it would look a little wonky.

Reply:
Oh also why i'm thinking of it --- you can do

```
<Button as={Link} to={tradeAssetLink} ... />

```

Not a huge deal but the button accepts the as prop.

Reply:
> This is prob gonna burn us for assets with really long symbols. 

Right! If we remove it for this reason on the Trade button, should we also revert the same symbol on the Buy/Sell button (for assets that are supported by ramps)? I didn't really consider symbols longer than 4-5 characters for both.

We could also not display the ticker if it is too long and keep it for usual shorter tickers... I kinda like to differentiate them from the navBar items as they both do something slightly different. 

I'll do some tests with various lengths and push something later! Thanks for the review

Reply:
Actually after testing different real symbols:

![image](https://user-images.githubusercontent.com/88804546/205382238-74cceb9c-8024-47b2-bc74-93b39317dfef.png)

![image](https://user-images.githubusercontent.com/88804546/205382309-31aad3b9-782a-49b0-aa05-93e2520a83d9.png)

It *degrades* actually pretty well I think.

Even if I (manually) force a ticker that is ultra long without word-break characters (space, hyphen, etc):

![image](https://user-images.githubusercontent.com/88804546/205382844-eb4ead7c-4e68-4aaa-a522-b03ea5ef7864.png)

At this stage we have bigger existing issues in other components like the TradeCard but it's out of the scope of this PR: 

![image](https://user-images.githubusercontent.com/88804546/205383608-777be73e-4882-4fa7-a93b-4edf8c511892.png)


So can we leave it like this @reallybeard  ?

Reply:
> Oh also why i'm thinking of it --- you can do
> 
> ```
> <Button as={Link} to={tradeAssetLink} ... />
> ```
> 
> Not a huge deal but the button accepts the as prop.

![image](https://user-images.githubusercontent.com/88804546/205384596-2162b375-9c18-48c6-810f-626eea60364f.png)

Seems like this syntax renders a `<a>` instead of `<button>` and it would require more styling to match the other buttons. If it's OK with you I'll leave the current syntax?

At first I thought about using `history.push` but it would require more import statements too 😓

Reply:
Yah I agree it wraps nicely which is good for translations, however its super redundant info on mobile. You have the asset name/symbol right above the buttons. I don't think it's needed here.

And yah doesn't matter too much on the button having a link inside it vs using the as prop. So good to me.

Reply:
OK, the symbol is removed from the Trade button.